src/main/java/fridayproject/Friday.java

```diff
@@ -0,0 +1,200 @@
+                    break;
+                }
+            } catch (Exception e) {
+                System.out.println("Error loading task: " + taskString);
+            }
+        }
+        scanner.close();
+        return tasks;
+    }
+
+    /**
+     * Main method to run the program.
+     * @param args
+     * @throws FridayException
+     * @throws IOException 
+     */
+    public static void main(String[] args) throws FridayException, IOException {
+        String greeting = "Hello! I'm Friday\nWhat can I do for you?\n";
+        String farewell = "Bye. Hope to see you again soon!";
+        ArrayList<Tasks> tasks = new ArrayList<>();
+        Scanner scanner = new Scanner(System.in); 
+        String line = "_________________________________________";
+        System.out.println(greeting + line);
+
+        try {
+            tasks = loadTasksFromFile();
+        } catch (IOException e) {
+            System.out.println("Error loading tasks from file: " + e.getMessage());
+            tasks = new ArrayList<>();
+        }
+        
+        while (true) {
+            String inputString = scanner.nextLine().trim();
+            System.out.println(line);
+            try {
+                System.out.println(line);
+                if (inputString.equals("bye")) {
+                    System.out.println(farewell);
+                    System.out.println(line);
+                    break;
+                } else if (inputString.startsWith("todo ")) {
+                    if (inputString.length() < 6) {
+                        throw new FridayException("I'm sorry, but I don't know what that means :(((\n Please enter a valid task description.");
+                    }
+                    Tasks todo = new Todo(inputString.substring(5)); 
+                    tasks.add(todo);
+                    saveTasksToFile(tasks);
+                    System.out.println("Got it. I've added this task:\n  " + todo.getTypeIcon() 
+                    + todo.toString() + "\nNow you have " + tasks.size() + " tasks in the list.");
+                    System.out.println(line);
+                } else if (inputString.startsWith("deadline")) {
+                    if (inputString.length() < 10) {
+                        throw new FridayException("I'm sorry, but I don't know what that means :(((\n Please enter a valid task description.");
+                    }
+                    String remainingInput = inputString.substring(inputString.indexOf(" ") + 1);
+                    String[] deadlineParts = remainingInput.split(" /by ");
+                    Tasks deadline = new Deadline(deadlineParts[0] , deadlineParts[1]);
+                    tasks.add(deadline);
+                    saveTasksToFile(tasks);
+                    System.out.println("Got it. I've added this task:\n  " + deadline.getTypeIcon() 
+                    + deadline.toString() + "\nNow you have " + tasks.size() + " tasks in the list.");
```

Comment:
Similar issue to lines 130-131? Lines wrapped should have an extra indent of 4 spaces based on the coding standard.